### ISSUE TITLE
Support for `input()`?

### ISSUE THREAD

**Issue by ceuk** [VSCodeVim/Vim#6591]
Hellooo 

I have a very simple vimscript function that I rely on quite often.  but it relies on an `input` from the user at the start. 

Is this currently supported via mappings or some other method?

If not would it be considered as a feature? 

One way to implement it fairly easily might be to:

a) Allow referencing of functions defined in a vimrc (when enabled) 
b) Support input() (i.e. open a dialogue in vscode and return the input)

Thanks for the great plugin :)

**Comment by J-Fields** [comment]
Yeah vimscript has only *very* limited support currently, unfortunately. More complete support (including `input`) is planned, but it's a big lift, so it'll probably be a while

**Comment by ceuk** [comment]
@J-Fields I haven't got loads of time at the moment but if at some point when you've planned the implementation your able to define a few small/medium tasks I could try help out. Just ping me on this issue :)
